packages/kit-bg/src/services/ServiceFirmwareUpdate/ServiceFirmwareUpdate.ts

```
@@ -362,6 +362,10 @@ class ServiceFirmwareUpdate extends ServiceBase {
 
     const currentTransportType =
       await this.backgroundApi.serviceSetting.getHardwareTransportType();
+
+    const isBleTransport =
+      currentTransportType === EHardwareTransportType.BLE;
```

ESLint

[error] Delete `⏎·····`

(prettier/prettier)